tablo/api.py

```diff
@@ -478,6 +478,9 @@ def deploy(self, request, **kwargs):
         except InternalError as e:
             logger.exception(e)
             raise ImmediateHttpResponse(HttpBadRequest('Error deploying file to database.'))
+        except Exception as e:
```

Comment:
There are some calls directly to the database that may throw non-InternalError exceptions. I was hitting some of those during this. Without wrapping all those in InternalError we have the chance of missing certain exceptions. Now, with the changes I have made here it lessens the chance of those exceptions, but doesn't eliminate them.